rotkehlchen/globaldb/cache.py

```diff
@@ -66,7 +66,7 @@ def globaldb_set_general_cache_values(
 def globaldb_delete_general_cache_values(
         write_cursor: DBCursor,
         key_parts: Iterable[str | GeneralCacheType],
-        values: tuple[str] | None = None,
+        values: list[str] | None = None,
```

Comment:
Was because I added this and used `(x for...)` here https://github.com/rotki/rotki/pull/7252/files#diff-ee958c2c0ab7f9ad214d7ac0f927b7d37e4cf2b6d200567d6ecee8578b802832R1998 and that was creating a generator but indeed I can just do `tuple(x for x...)`

Reply:
but need to change it to `tuple[str, ...]` since as it is now is wrong